app/controllers/plot_plants_controller.rb

```diff
@@ -0,0 +1,9 @@
+class PlotPlantsController < ApplicationController
+  
+  def destroy
+    @plot = Plot.find(params[:plot_id])
+    @plot.plants.find(params[:id]).destroy
+    redirect_to "/plots"
+  end
```

Comment:
This is very close. For this US, we want to delete the record from join table. We're used to adding to the join table, this US is the opposite now.